about.html

```diff
@@ -12,53 +12,85 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
+  <div class="container about-page">
+  <div>
```

Comment:
Here you have a <div> surrounding your navigation and also <nav> surrounding all your navigation elements. Since you are not using the <div> to add any semantic meaning to the page or using it for CSS styling purposes it is not needed in this case.